modules/search/instant-search/components/search-results.jsx

```diff
@@ -166,15 +166,15 @@ class SearchResults extends Component {
 				aria-live="polite"
 				className="jetpack-instant-search__search-results"
 			>
-				<a
+				<button
```

Comment:
An `a` with `role="button"` does not provide all the same keyboard events, like being able to activate the button using either the ENTER or SPACE keys. We could reimplement those ourselves, but it makes sense to just use the `button` element if we can.